### PR TITLE
Updated footer

### PR DESCRIPTION
**Pull request by PepperAddict**
##Issue Number: #148  

I wasn't sure if I needed to create a branch for this or not. 
## Description

I updated the names in the footer JSON using graphQL's data. I made the link to read the name that was given or the username. Let me know if this is not something you wanted.

I changed the footer to use ul instead of div. 
I fixed the style just a little bit to make it look like before. 
Also removed min width from the style for #105 

##Do any new issues need to be created as a result of this PR"?
I don't think so. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

🚨Please review the [contribution guidelines](https://github.com/gwg-women/gwg-women-techmakers/wiki/Contribution-Guidelines) to this repository.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Created a new branch from 'dev' and PR is targeting the project's 'dev' branch.
- [x] The issue can be closed.

 Provide a descriptive branch name and once the pull request is approved and merged to dev, the branch can be deleted.

💔Thank you!

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -230,12 +128,12 @@ export class Footer extends Component {
...
   render(){
     const {members} = this.state;
-    let together = members.join(" | ");
+    let together = members.join(" ");
     
     return (
       <div>
-        <a href="https://github.com/gwg-women/gwg-women-techmakers"> Grow with Google Project</a>: <div dangerouslySetInnerHTML={{ __html: together}} />
-      </div>
+        <h1 className="gwg-link"><a href="https://github.com/gwg-women/gwg-women-techmakers"> Grow with Google Project</a></h1> <ul dangerouslySetInnerHTML={{ __html: together}} />
```
Comment (digilou) [inline src/components/Footer.js]: If this is going to be a heading, it should be an h2. H1 reserved for title of page/application.

File: src/css/index.css [hunk 2]
```diff
@@ -139,40 +176,67 @@ h3 {
...
+}
+
+.footer a:hover, .footer a:focus {
+  background: white;
+  color: black;
+  border-radius: 10px;
+}
+
+.footer ul, .footer li {
+  float: left;
```
Comment (digilou) [inline src/css/index.css]: After all your awesome grid work @PepperAddict?! 😃 
I've been experimenting with flexbox over floats lately. Flex is great for lists like this, and you don't have to worry about fallbacks, as it's supported more widely than grid.
But overall, this is just my personal suggestion. Floats are not an issue, in and of themselves.

**Review by digilou — CHANGES_REQUESTED** [review]

**Comment by digilou** [comment]
Thanks for all your work @PepperAddict! This is a helpful update.

**Comment by sonalikatara** [comment]
@PepperAddict , you were right about creating a new branch for this work. Each time you start working on a new issue, you should create a new branch with a descriptive name.
This code looks good to merge once we get the changes suggested above. Thank you for the good work.

**Review by digilou — APPROVED** [review]

**Review by khusbuchandra — APPROVED** [review]
Thanks for the good work @PepperAddict

**Pull request merged**